src/bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -676,10 +676,7 @@ def process_columns_contents(content: pb.Series) -> dict[str, t.Any]:
             with ThreadPoolExecutor(max_workers=10) as executor:
                 futures = executor.map(process_columns_contents, field.columns)
                 data.extend([i for i in list(futures)])
-            dataframe = pd.DataFrame(
-                dict(zip(field.column_names, data)),
-                columns=t.cast(t.List[str], field.column_names),
-            )
+            dataframe = pd.DataFrame(dict(zip(field.column_names, data)))
```

Comment:
I believe when I first write this, the behaviour of DataFrame then ircc requires to specify the columns regardless the input type.

Reply:
In `pandas>=1` at least `columns` is not needed if the data passed is a `dict`: https://pandas.pydata.org/pandas-docs/version/1.0/reference/api/pandas.DataFrame.html